spec/controllers/bookmarks_controller_spec.rb

```diff
@@ -51,6 +51,7 @@
       user.bookmarks.create(question: question_3)
       sign_in user
       allow(controller).to receive(:current_user).and_return(user)
+      # allow(BookmarksExportService).to_receive(:export).with(format).and_return(export_service)
```

Comment:
if this isn't being used could we remove it?

Reply:
I meant to... didn't I? 

There was a LOT of stuff... not surprised if I missed some!